radpad.sh

```diff
@@ -6,6 +6,7 @@
 # so only query for the first match
 alias rpexec="docker exec -it `docker ps | grep radpad/ | awk 'NR==1{ print $1 }'`"
 alias rpexec_notty="docker exec `docker ps | grep radpad/ | awk 'NR==1{ print $1 }'`"
+alias rplogs="docker logs -f `docker ps | grep radpad/ | awk 'NR==1{ print $1 }'`"
```

Comment:
This is what I get for reviewing a PR before coffee. A better solution would be to alias the `docker ps | grep radpad/ | awk 'NR==1{ print $1 }'` portion (and was my intent in the original comment).